app/parsers/bulkrax/oai_dc_parser.rb

```diff
@@ -61,6 +61,11 @@ def import_fields
 
     delegate :list_sets, to: :client
 
+    # TODO: not yet supported
```

Comment:
the empty methods in this file are to override the now defaults in application_parser.rb

Reply:
Instead of "unsetting" defaults, should the bagit parser just inherit from the csv parser? (see [related comment](https://github.com/samvera-labs/bulkrax/pull/571#discussion_r902779855))